docker/build/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Images build from this Dockerfile are used in Wercker to build
 # our webiste and documentation
 
-FROM ubuntu
+FROM debian:stable
```

Comment:
Why'd we switch from Ubuntu to Debian here? It's not really an issue, just curious.

Reply:
stability. we didn't have a set version of ubuntu.... not sure what was used for the old version.